lib/cartopy/io/ogc_clients.py

```diff
@@ -807,6 +869,36 @@ def _to_shapely_geoms(self, response):
                 data = self._find_polygon_coords(node, find_str)
                 points_data.extend(data)
 
+        # Try to get other geometries from other servers than
+        # http://mapserver.gis.umn.edu/mapserver
```

Comment:
Is there a way to add an extra line or two above to make that loop work for your case? i.e. can you add something above to change the `tree.findall()` call like updating the `_MAP_SERVER_NS` or just adding an extra block to find all of your geom tags?

This just reads as a lot of copied code, so another option would be to make a function that both loops could use too. Something like this to update your geometry lists that could be re-used in both loops.

`linear_rings_data, linestrings_data, point_data = _get_geometries(node, linear_rings_data, linestrings_data, point_data)

Reply:
O.K. I was trying to not change existing code. Now it is included in the same loop.